.github/workflows/pytest_otel.yml

```diff
@@ -31,3 +31,8 @@ jobs:
         run: make -C resources/scripts/pytest_otel test
       - name: it-test
         run: make -C resources/scripts/pytest_otel it-test
+      - uses: rainbow-duck-games/unity-test-publisher@82614fdbce4f519c5f75b24511d94250af6c04d9
+        if: ${{ always() }} # Avoid skipping on failed tests
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          reportPaths: resources/scripts/pytest_otel/**/junit-*.xml
```

Comment:
no strong opinion, though I also researched other publishers such as:
- EnricoMi/publish-unit-test-result-action@f00bb47e1d89c6d98071d1b69e16a2b63c7fbfc1
- dorny/test-reporter@v1

as they suggested to use another event instead running within the default workflow, as per https://github.com/EnricoMi/publish-unit-test-result-action#support-fork-repositories-and-dependabot-branches